src/sqerl_client.erl

```diff
@@ -257,3 +246,30 @@ read_statements_from_config() ->
             error_logger:error_report(Msg),
             error(Msg)
     end.
+
+
+%% Helper function to report and error
+
+log_and_error(Msg) ->
+    error_logger:error_report(Msg),
+    error(Msg).
+
+maybe_log(true, QueryOrName, Args) ->
+    error_logger:info_msg("(~p) Executing statement ~p with args ~p~n", [self(), QueryOrName, Args]);
+maybe_log(_, _, _) ->
+    ok.
+
+maybe_log(true, Results) ->
+    error_logger:info_msg("(~p) Result: ~p~n", [self(), Results]);
+maybe_log(_, _) ->
+    ok.
+
+pool_config(Pool) ->
+    % Support backward compatibility for configurations that do not specify
+    % 'databases'.
+    case envy:get(sqerl, databases, none, list) of
+        none ->
```

Comment:
Would a more special atom be better here? I mean `none` is an unlikely db/pool name, but we could easily narrow the namespace further.